azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -883,23 +891,51 @@ while ($workflowStatus.status -ne "completed") {
 
 foreach ($cluster in $AgConfig.SiteConfig.GetEnumerator()) {
     Start-Job -Name gitops -ScriptBlock {
-        $AgConfig = $using:AgConfig
-        $cluster = $using:cluster
-        $namingGuid = $using:namingGuid
-        $resourceGroup = $using:resourceGroup
-        $appClonedRepo = $using:appClonedRepo
+        
+        Function Get-GitHubFiles ($githubApiUrl, $folderPath, [Switch]$excludeFolders) {
```

Comment:
should we move this function definition to the psprofile.ps1 file ?

Reply:
I think we should move the function out of the logon script, but we're talking about doing a larger refactoring and I believe that would be the ideal time to move it. Right now, it's only being used by code in the immediate area.